go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/CosmWasm/wasmvm v1.0.1
 	github.com/armon/go-metrics v0.4.1
 	github.com/btcsuite/btcd v0.22.1
-	github.com/cosmos/cosmos-sdk v0.45.10
+	github.com/cosmos/cosmos-sdk v0.47.10
```

Comment:
Should be reversed since we actually meant to upgrade sei-cosmos, not cosmos-sdk